includes/class-webmention-receiver.php

```diff
@@ -387,6 +438,39 @@ public static function post( $request ) {
 		return new WP_REST_Response( $return, 200 );
 	}
 
+	/**
```

Comment:
should be also part of the vouch class?

Reply:
Back to the question I asked re storage.

Reply:
OK, will think about that, but then I have one more question: Why not directly set the comment-meta in line 318 and following and move it above the commentdata setter in line 316?

Reply:
Because there might not be a vouch parameter? It is optional.

Reply:
But this is what you also check in line 318...

Reply:
why not:

```php
if ( isset( $params['vouch'] ) ) {
	$comment_meta['vouch'] = esc_url( urldecode( $params['vouch'] ) );
}
```

Reply:
I misunderstood...you are right.

Reply:
like you do with the date: https://github.com/pfefferle/wordpress-webmention/pull/181/files#diff-4a405db849c60f60b9e32bde3bfe0096R308